mp/src/game/shared/momentum/util/engine_patch.h

```diff
@@ -0,0 +1,56 @@
+//------------------------------------------------------------------------------------
+// Functions used to find patterns of bytes in the engine's memory to hook or patch
+//-----------------------------------------------------------------------------------
+#pragma once
+
+class CEngineBinary : CAutoGameSystem
```

Comment:
We typically do public inheritance here:
```suggestion
class CEngineBinary : public CAutoGameSystem
```